package.json

```diff
@@ -97,6 +97,7 @@
     "sockjs-client": "^1.5.1",
     "standard-version": "^9.3.0",
     "style-loader": "^2.0.0",
+    "strip-ansi-v6": "npm:strip-ansi@^6.0.0",
```

Comment:
We need to migrate our test on ECMA modules, but let's do it late

Reply:
Will take care 👍

Reply:
Ideally we should do it for all our repos in webpack/webpack-contrib except webpack itself